spec/services/workflows/non_passported_workflow_spec.rb

```diff
@@ -5,20 +5,23 @@ module Workflows
     let(:assessment) do
       create :assessment, :with_capital_summary, :with_disposable_income_summary,
              :with_gross_income_summary,
-             applicant:, proceedings: [%w[SE003 A]]
+             applicant:, proceedings: proceeding_types.map { |p| [p, "A"] }, level_of_representation: level_of_work
     end
 
     before do
       assessment.proceeding_type_codes.each do |ptc|
-        create :capital_eligibility, upper_threshold: 20_000, capital_summary: assessment.capital_summary, proceeding_type_code: ptc
         create :gross_income_eligibility, gross_income_summary: assessment.gross_income_summary, proceeding_type_code: ptc
         create :disposable_income_eligibility, disposable_income_summary: assessment.disposable_income_summary,
                                                lower_threshold: 500,
                                                proceeding_type_code: ptc
       end
+      Creators::CapitalEligibilityCreator.call(assessment)
     end
 
     describe ".call" do
+      let(:level_of_work) { "certificated" }
```

Comment:
I think for consistency this should be `level_of_representation` (until we rename the whole thing to `level_of_help` in the near future). To the best of my knowledge we don't use "level of work" anywhere else?